lambda/input/src/main/scala/Module.scala

```diff
@@ -1,8 +1,14 @@
+import akka.actor.ActorSystem
 import com.google.inject.AbstractModule
+import com.theguardian.multimedia.archivehunter.common.clientManagers.{ESClientManager, ESClientManagerImpl}
 import com.theguardian.multimedia.archivehunter.common.{ArchiveHunterConfiguration, ArchiveHunterConfigurationExt}
+import akka.stream.Materializer
 
-class Module extends AbstractModule {
+class Module(system:ActorSystem, mat:Materializer) extends AbstractModule {
   override def configure(): Unit = {
+    bind(classOf[ActorSystem]).toInstance(system)
```

Comment:
The reason for the inclusion ActorSystem is because it is required by JobModelDAO and LightboxEntryDAO. We would have the replace them with something else to get data in and out of the models to remove ActorSystem.